Pod/Classes/URBNAccordionDataSourceAdapter.m

```diff
@@ -10,7 +10,7 @@
 
 @interface URBNAccordionDataSourceAdapter ()
 
-@property (nonatomic, strong) NSMutableArray *sections;
+@property (nonatomic, strong) NSArray *sections;
```

Comment:
I’m just keeping this a private property and adding an `-allSections` to mimic `- allItems`